packages/manager/modules/overthebox/src/overthebox/autoconfigure/dhcp/template.html

```diff
@@ -0,0 +1,290 @@
+                            ng-model="$ctrl.config.leaseDuration"
+                        />
+                    </oui-field>
+                    <oui-field
+                        class="ml-2"
+                        label="{{:: 'overTheBox_autoconfigure_dhcp_offset' | translate}}"
+                        label-popover="{{:: 'overTheBox_autoconfigure_dhcp_offset_description' | translate}}"
+                    >
+                        <input
+                            class="oui-input"
+                            name="offset"
+                            type="number"
+                            required
+                            ng-model="$ctrl.config.offset"
+                        />
+                    </oui-field>
+                    <oui-field
+                        class="ml-2"
+                        label="{{:: 'overTheBox_autoconfigure_dhcp_pool_size' | translate}}"
+                    >
+                        <input
+                            class="oui-input"
+                            name="poolSize"
+                            type="number"
+                            required
+                            ng-model="$ctrl.config.poolSize"
+                        />
+                    </oui-field>
+                    <oui-field
+                        class="ml-2"
+                        label="{{:: 'overTheBox_autoconfigure_priority' | translate}}"
+                    >
+                        <input
+                            class="oui-input"
+                            name="priority"
+                            type="number"
+                            required
+                            ng-model="$ctrl.config.priority"
+                        />
+                    </oui-field>
+                    <oui-button
+                        class="ml-2 mb-2"
+                        variant="primary"
+                        type="submit"
+                        on-click="$ctrl.createConfig()"
+                    >
+                        <span
+                            data-translate="overTheBox_autoconfigure_submit"
+                        ></span>
+                    </oui-button>
+                    <oui-button
+                        class="ml-2 mb-2"
+                        variant="primary"
+                        on-click="$ctrl.cancelCreateConfig()"
+                    >
+                        <span
+                            data-translate="overTheBox_autoconfigure_cancel"
+                        ></span>
+                    </oui-button>
+                </div>
+            </form>
+        </div>
+        <oui-button
+            variant="primary"
+            data-on-click="$ctrl.addConfig()"
+            class="mb-2 d-flex justify-content-center"
+            data-ng-if="$ctrl.displayAddConfigButton"
+        >
+            <span
+                data-translate="overTheBox_autoconfigure_dhcp_config_add"
+            ></span>
+        </oui-button>
```

Comment:
**question:** Why not prefer using `<oui-datagrid-topbar>`?
see: https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-datagrid-webcomponent--with-topbar